### DISCUSSION TITLE
Is there a dashboard for Conductor metrics?

### DISCUSSION THREAD

**Discussion by emr17** [Netflix/conductor#3515] [General]
I have activated the metrics for the conductor through prometheus, but I want to visualize these metrics. I saw that grafana was used for this, but I could not find a dashboard prepared according to metrics. There is a dashboard that was prepared 3 years ago, but it does not work, I guess because it is not up to date. Can you help if there is a currently working dashboard?

**Comment by DukeDai** [comment]
one prometheus query example, but would be better there is template. 
`sum by(workflowName) (rate(workflow_running{env="$env", app="conductor-server"}[$__rate_interval]))`
